misaligned.js

```diff
@@ -1,16 +1,20 @@
 const {expect} = require('chai')
+const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
+const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
 
-function print_color_map() {
-    const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
-    const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
+function print_color_map(majorColors,minorColors) {
     for (let i = 0; i < majorColors.length; i++) {
         for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
+            console.log(`${i * 5 + j + 1} | ${majorColors[i]} | ${minorColors[j]}`);
         }
-    }
+    }    
+}
+
+function print_color_length(majorColors,minorColors){
     return majorColors.length * minorColors.length;
 }
 
-result = print_color_map();
-expect(result).equals(25);
+result = print_color_map(majorColors,minorColors);
+printColorLength = print_color_length(majorColors,minorColors);

```

Comment:
The test passes even when the separators are not aligned. Absence of an alignment test means that manual inspection of alignment is needed every time something changes.
Try a test for alignment
Is it hard? Why not use off-the shelf functionality like console.table?